provider/clusterrosa/classic/cluster_rosa_classic_resource.go

```diff
@@ -108,8 +108,18 @@ func (r *ClusterRosaClassicResource) Schema(ctx context.Context, req resource.Sc
 				},
 			},
 			"name": schema.StringAttribute{
-				Description: "Name of the cluster. Cannot exceed 15 characters in length. " + common.ValueCannotBeChangedStringDescription,
+				Description: "Name of the cluster. Cannot exceed 54 characters in length. " + common.ValueCannotBeChangedStringDescription,
 				Required:    true,
+				Validators: []validator.String{
+					stringvalidator.LengthAtMost(54),
+				},
+			},
+			"domain_prefix": schema.StringAttribute{
+				Description: "The domain prefix is optionally assigned by the user." +
+					"It will appear in the Cluster's domain when the cluster is provisioned. " +
+					"If not supplied, it will be auto generated. It cannot exceed 15 characters in length" + common.ValueCannotBeChangedStringDescription,
+				Optional: true,
+				Computed: true,
 				Validators: []validator.String{
 					stringvalidator.LengthAtMost(15),
```

Comment:
Shouldn't it be changed here as well?

Reply:
This is for the domain_prefix. It needs to be <= 15 chars.